ts_src/index.ts

```diff
@@ -174,23 +195,62 @@ export function entropyToMnemonic(
 
   const bits = entropyBits + checksumBits;
   const chunks = bits.match(/(.{1,11})/g)!;
-  const words = chunks.map(
-    (binary: string): string => {
+  const wordsAsBuffers = chunks.map(
+    (binary: string): Buffer => {
       const index = binaryToByte(binary);
-      return wordlist![index];
+      wordlist = wordlist || [];
+      return Buffer.from(wordlist[index], 'utf8');
     },
   );
 
-  return wordlist[0] === '\u3042\u3044\u3053\u304f\u3057\u3093' // Japanese wordlist
-    ? words.join('\u3000')
-    : words.join(' ');
+  const bufferSize = wordsAsBuffers.reduce(function(
```

Comment:
This causes a lint error on `yarn run lint`, because the current lint rules prefer arrow functions.

Another example down below in the `wordsAsBuffers.reduce(` call

Reply:
Also this function should have types declared explicitly. That sets off another rule.